### ISSUE TITLE
auto-merge cats

### ISSUE THREAD

**Issue by gogogo1112** [andbolt/catizen_clicker#1]
do you have a script to automatically crossbreed cats?

**Comment by andbolt** [comment]
Hi. No, the auto-merging is not implemented here. However I might add it in the future versions.

**Comment by gogogo1112** [comment]
> Hi. No, the auto-merging is not implemented here. However I might add it in the future versions.

Well, I'd be very grateful

**Comment by gogogo1112** [comment]
> Hi. No, the auto-merging is not implemented here. However I might add it in the future versions.

Are you able to write a script that will automatically crossbreed kitties?

**Comment by andbolt** [comment]
Yes, I am able, however it is not of top priority right now.  Please wait for an update.
